.azure/azure-pipelines.ci.yml

```diff
@@ -224,6 +218,25 @@ stages:
       platform: linux
       tls: openssl
 
+- stage: codecoverage
+  displayName: CodeCoverage
+  dependsOn:
+  - build_windows
+  - build_linux
```

Comment:
We don't actually depend on Linux right?

Reply:
not at this time, no. if we add support for linux, we could.